expfmt/text_parse.go

```diff
@@ -190,28 +195,38 @@ func (p *TextParser) startComment() stateFn {
 	if p.readTokenAsMetricName(); p.err != nil {
 		return nil // Unexpected end of input.
 	}
-	if p.currentByte == '\n' {
+	p.setOrCreateCurrentMF()
+
+	// In case we are parsing a HELP comment but no value has been
+	// supplied we make sure that we set the string to empty
+	if keyword == KeywordHelp && p.currentMF.Help == nil {
+		p.currentMF.Help = proto.String("")
+	}
+
+	if isNewline(p.currentByte) {
 		// At the end of the line already.
 		// Again, this is not considered a syntax error.
 		return p.startOfLine
 	}
+
```

Comment:
Here you are even introducing vertical whitespace in unaffected parts. Obviously, it's a matter of taste and different coders have different styles, but the golden rule is to adapt newly added code to the style around it (and not the other way around).